python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -12,7 +12,7 @@
 
 
 def columns_from_dataframe(df):
-    cols = [sr._column for sr in df._cols.values()]
+    cols = list(df._data.values())
```

Comment:
Doesn't need to be a list?

Reply:
The caller isn't able to work with an `iter` here. Hopefully this will all go away with the groupby refactor anyway